iphone/Classes/TiUIActivityIndicator.m

```diff
@@ -287,16 +287,31 @@ - (void)updateConstraints
                                                                    attribute:NSLayoutAttributeCenterY
                                                                   multiplier:1
                                                                     constant:0]];
-        [backgroundView addConstraint:[NSLayoutConstraint constraintWithItem:indicatorView
-                                                                   attribute:NSLayoutAttributeCenterY
-                                                                   relatedBy:NSLayoutRelationEqual
-                                                                      toItem:backgroundView
-                                                                   attribute:NSLayoutAttributeCenterY
-                                                                  multiplier:1
-                                                                    constant:0]];
+        // FIX : TIMOB-20513
+        /*
+         [backgroundView addConstraint:[NSLayoutConstraint constraintWithItem:indicatorView
+         attribute:NSLayoutAttributeCenterY
+         relatedBy:NSLayoutRelationEqual
+         toItem:backgroundView
+         attribute:NSLayoutAttributeCenterY
+         multiplier:1
+         constant:0]];
+         */
```

Comment:
Remove the comments please. Old code can be revised using the git-blame tool as discussed.